spiffworkflow-backend/src/spiffworkflow_backend/routes/process_api_blueprint.py

```diff
@@ -77,6 +78,14 @@ def process_list() -> Any:
     return SpecReferenceSchema(many=True).dump(references)
 
 
+def process_callers(bpmn_process_identifier: str) -> Any:
+    callers = ProcessCallerService.callers(bpmn_process_identifier)
+    references = (
+        SpecReferenceCache.query.filter_by(type="process").filter(SpecReferenceCache.identifier.in_(callers)).all()
+    )
+    return SpecReferenceSchema(many=True).dump(references)
```

Comment:
we're trying to move away from marshmallow schemas in favor of dataclasses, etc, for serialization. no action required here, since i know SpecReferenceCache already had the Schema, but just for new models for the future.